### PR TITLE
Fix typos on 17 files.

### PR DESCRIPTION
**Pull request by Sajjon**
Fix typos in 17 files

### CODE REVIEW

File: bip-0158/gentestvectors.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ var (
 		{49291, "Tx pays to empty output script"},
 		{180480, "Tx spends from empty output script"},
 		{926485, "Duplicate pushdata 913bcc2be49cb534c20474c4dee1e9c4c317e7eb"},
-		{987876, "Coinbase tx has unparseable output script"},
+		{987876, "Coinbase tx has unparsable output script"},
```
Comment (vostrnad) [inline bip-0158/gentestvectors.go:40]: "unparseable" is not a typo but a valid alternative spelling.

**Review by vostrnad — APPROVED** [review]
ACK mod one false positive.

**Review by murchandamus — APPROVED** [review]
LGTM, although "queriable/queryable" and "unparseable/unparsable" both seem acceptable

ACK 1eefea0456d661b4c51f3a40b7262d6a99161edd

File: bip-0158/gentestvectors.go (hunk 1 above)
Comment (Sajjon) [reply bip-0158/gentestvectors.go:40]: Indeed, so somewhat false positive, let me know if I should revert.

**Pull request merged**